Query and update vet clinic database

ismayito opened the pull request:
### Updating the table, Querying, and using the JOIN to query tables
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
#### Summary 
_____________________________________________________________________________________________________
In this, I added the following:
-  I added more tables the owners table and the species table
- I altered the animal table and dropped the species column
- I added the foreign keys into the animals table referencing both the owners table and species table
- I also queried the tables to perform complex calculations using the JOIN and GROUP BY
![Screenshot from 2023-10-25 15-55-14](https://github.com/ismayito/vet-clincic-database/assets/47735113/289278dc-2e7c-4e1f-9256-cd8c325036e7)
![Screenshot from 2023-10-25 16-05-52](https://github.com/ismayito/vet-clincic-database/assets/47735113/c62890d5-0053-4b35-989c-111ae90e2937)
![Screenshot from 2023-10-25 16-11-19](https://github.com/ismayito/vet-clincic-database/assets/47735113/ffd54ea9-e5c6-4a93-8511-93f2877f6b29)
![Screenshot from 2023-10-25 16-20-29](https://github.com/ismayito/vet-clincic-database/assets/47735113/563ab151-0324-45f1-9a7e-46abceada634)
![Screenshot from 2023-10-25 16-37-50](https://github.com/ismayito/vet-clincic-database/assets/47735113/8724134f-a87e-40e6-a6d0-f08bafcf269b)
![Screenshot from 2023-10-25 18-06-04](https://github.com/ismayito/vet-clincic-database/assets/47735113/444532eb-591b-4ed3-8c58-ce6e5ba88b36)
![Screenshot from 2023-10-25 18-09-37](https://github.com/ismayito/vet-clincic-database/assets/47735113/57a52c54-9240-497b-b739-408f52bd66db)
![Screenshot from 2023-10-26 17-20-16](https://github.com/ismayito/vet-clincic-database/assets/47735113/3959cfdb-7b25-40b8-9530-ce0d13779140)
![Screenshot from 2023-10-26 17-25-30](https://github.com/ismayito/vet-clincic-database/assets/47735113/e012986a-a686-4252-9c8e-e141521018d2)
![Screenshot from 2023-10-26 17-27-46](https://github.com/ismayito/vet-clincic-database/assets/47735113/98d1a210-5599-4db0-8df0-e2aa4dff8f9a)

![Screenshot from 2023-10-25 20-09-41](https://github.com/ismayito/vet-clincic-database/assets/47735113/b6cc8509-7f00-4f6b-951b-ef2bd9ee9aa2)
![Screenshot from 2023-10-25 20-56-21](https://github.com/ismayito/vet-clincic-d
![Screenshot from 2023-10-26 12-56-30](https://github.com/ismayito/vet-clincic-database/assets/47735113/907d63f1-ad49-4e75-8b5f-6e9f1197e4f0)
![Screenshot from 2023-10-26 12-59-24](https://github.com/ismayito/vet-clincic-database/assets/47735113/28ec1d69-e1ed-410f-9fd4-aeb828ef83ce)
![Screenshot from 2023-10-26 13-00-25](https://github.com/ismayito/vet-clincic-database/assets/47735113/1db21cdf-6de0-48e6-b492-c55b5de10858)
![Screenshot from 2023-10-26 13-03-04](https://github.com/ismayito/vet-clincic-database/assets/47735113/1d671412-a1d8-4c8d-a362-fc87ce7b419d)
![Screenshot from 2023-10-26 13-03-58](https://github.com/ismayito/vet-clincic-database/assets/47735113/667f145c-4e79-4a54-b142-21c933a64295)
![Screenshot from 2023-10-26 13-06-57](https://github.com/ismayito/vet-clincic-database/assets/47735113/99f35b2a-b572-4ffc-a014-a524922fe10d)
![Screenshot from 2023-10-26 13-07-44](https://github.com/ismayito/vet-clincic-database/assets/47735113/64233be9-d562-49df-84e6-2ce59c7347c2)
atabase/assets/47735113/3c81f0df-3d27-48ee-bd75-33bc3ba8b9be)
![Screenshot from 2023-10-25 21-04-47](https://github.com/ismayito/vet-clincic-database/assets/47735113/09e91709-a9c7-4d20-adb3-52a296dd15d4)

ismayito committed:
Query and update vet clinic database

mikethreels commented on queries.sql:
```diff
@@ -54,4 +54,27 @@ UPDATE ANIMALS SET species='Pokemon' WHERE NAME='Squirtle';
+//Write queries (using JOIN) to answer the following questions
+//:What animals belong to Melody Pond?
+SELECT name FROM ANIMALS JOIN OWNERS ON ANIMALS.owner_id=OWNERS.id WHERE TRIM(OWNERS.full_name)='Melody Pond';
+//List of all animals that are pokemon (their type is Pokemon).
+SELECT ANIMALS.name FROM ANIMALS JOIN SPECIES ON ANIMALS.species_id=SPECIES.id WHERE ANIMALS.species_id=1;
```
- [ ] This query returns all the Digimon instead of Pokemon, this is most likely due to a small mistake in the `data.sql` file where you didn't set the value for Pokemon species.
https://github.com/ismayito/vet-clincic-database/blob/624048eab57177ba5b82c8bfe5feb6c7b0fd1856/data.sql#L29-L32

mikethreels reviewed (changes requested):
Hi ismayito, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job updating the tables.

## Required Changes ♻️
- [ ] Most of the screenshots you've added are for updating the tables and data, however, we are missing a lot of the screenshots for the queries, please make sure to include the screenshots for all the queries.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ismayito committed:
fix review changes

topeogunleye commented:
@ismayito, I am your code reviewer, and this is a comment, not a review. please kindly take note that you haven't done the changes requested by the previous code reviewer.

<img width="573" alt="image" src="https://github.com/ismayito/vet-clincic-database/assets/52934128/8c4ab8b6-64d8-4f3c-9115-d3b9a4011fe9">

topeogunleye reviewed (changes requested):
Hi @ismayito, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✅ Pokemon Species Setting: Great job on updating the data.sql file. I can now see the Pokemon as intended! Pikachu would be proud! ⚡

👍 Efficiency and Clarity: Once again, the efficiency and clarity of your queries are commendable. Your comments continue to make the code easily understandable.

 ## Required Changes ♻️
---

Hi @ismayito,

Upon reviewing your recent submission, I noticed that the required changes from the previous code review have not been addressed completely. While you've provided numerous screenshots for updating tables and data, we are still missing important screenshots that demonstrate the queries in action.

As previously mentioned by @mikethreels, it's crucial for us to see all the queries visually, especially in the context of an educational review. This aids both in understanding the flow and the correctness of your work.

Please refer to the earlier comments left under the review for more specifics.

Thank you for your attention to this matter. Once you've made the necessary additions, please resubmit your work, and I'll be happy to take another look.

Best,
Temitope

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ismayito committed:
fix review changes

skyv26 commented on data.sql:
```diff
@@ -15,3 +15,28 @@ VALUES('Charmander', '2020-02-08', 0, true, -11),
+
+//Modify your inserted animals so it includes the species_id value:
+//If the name ends in "mon" it will be Digimon
+//All other animals are Pokemon
+UPDATE ANIMALS SET species_id=2 WHERE TRIM(NAME) LIKE '%mon';
+//Sam Smith owns Agumon.
```
- [ ] Well done @ismayito, you did a great job by adding the query to update the animals but it seems like you removed the query for updating the `species_id = 1` animals ends with %mon in name, now with an above-selected query, in the database all animals species_id will set to `2`. But according to the project requirement that I also have added after this comment as a reference, you should set the animal species_id with 1 (Digimon), whose name ends with `%mon` and the rest of the animals should be set to (pokemon) `species_id` with 2. Don't worry I have added the bullet points with pseudo below. In short, both your queries should be in the `data.sql` file and the PR description with screenshots.

### Requirement

![image](https://github.com/ismayito/vet-clincic-database/assets/66476329/c0885738-00b2-4815-8971-b4f2f2ea25d9)

### Pseudo

1. Set the animals species_id = 1 where the name of the animals ends with "%mon" which means the last three characters are mon. (It will update the majority of the animals)
2. Set the rest of the animals with species_id = 2, where animals species_id is null. (Those animals will get the species_id = 2 which does contain the default value `Null` in the table)

skyv26 reviewed (changes requested):
## Hi @ismayito, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

## STATUS: REQUIRE CHANGES 🔴 

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOGYyYWNkMjRkY2JjYjg1MjY1M2NjYTNkNWNiNTQwN2EyZGZmNWM5YSZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/m3864rBwwBTKMAbpn3/giphy.gif)

## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implemented the features as per the previous code reviewer's request ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   <img src="https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white" alt="@skyv26" />
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   <img src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white" alt="linkedin-skyv2022" />
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   <img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white" alt="@vrma_aakash" />
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   <img src="https://img.shields.io/badge/Instagram-E4405F?style=for-the-badge&logo=instagram&logoColor=white" alt="@skybrel" />
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   <img src="https://user-images.githubusercontent.com/66476329/243158690-61eb1021-fbb6-464d-b841-a09c4caa156e.png" alt="@skybrel" />
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ismayito commented:
Hello unless otherwise but i updated the SQL and everything is fine


On Thu, Oct 26, 2023 at 3:28 PM Aakash Verma ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> Hi @ismayito <https://github.com/ismayito>,
>
> Great work on making the changes requested by a previous reviewer 👏
> You've done well implementing some of the requested changes, but some are
> still not addressed yet.
> STATUS: REQUIRE CHANGES 🔴
>
>
> <https://camo.githubusercontent.com/7cb34c1f1afdb9d140a7831d0b08fbde98644d3dbb133fdcf4e69aeecaef2189/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784f47597959574e6b4d6a526b59324a6a596a67314d6a59314d324e6a59544e6b4e574e694e5451774e3245795a475a6d4e574d3559535a6c634431324d563970626e526c636d35686246396e61575a7a583264705a6b6c6b4a6d4e305057632f6d333836347242777742544b4d4162706e332f67697068792e676966>
> Highlights
>
>    - Good *PR* title ✅
>    - Descriptive *PR* description ✅
>    - Implemented the features as per the previous code reviewer's request
>    ✅
>
> Required Changes ♻️
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you take them into
> account as they can make your code better.*
> Cheers, and Happy coding!👏👏👏 ⚠️ WARNING ⚠️
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> Have any doubt ❓
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag @skyv26 <https://github.com/skyv26> in your
> question so I can receive the notification. You can also follow me on the
> below platforms
> [image: @skyv26] <https://github.com/skyv26> [image: linkedin-skyv2022]
> <https://www.linkedin.com/in/skyv2022/> [image: @vrma_aakash]
> <https://twitter.com/vrma_aakash> [image: @skybrel]
> <https://www.instagram.com/skybrel/> [image: @skybrel]
> <https://www.frontendmentor.io/profile/skyv26>
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think the code review was unfair, you can request a
> second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In data.sql
> <https://github.com/ismayito/vet-clincic-database/pull/3#discussion_r1373085181>
> :
>
> > +//All other animals are Pokemon
> +UPDATE ANIMALS SET species_id=2 WHERE TRIM(NAME) LIKE '%mon';
> +//Sam Smith owns Agumon.
>
>
>    - Well done @ismayito <https://github.com/ismayito>, you did a great
>    job by adding the query to update the animals but it seems like you removed
>    the query for updating the species_id = 1 animals ends with %mon in
>    name, now with an above-selected query, in the database all animals
>    species_id will set to 2. But according to the project requirement
>    that I also have added after this comment as a reference, you should set
>    the animal species_id with 1 (Digimon), whose name ends with %mon and
>    the rest of the animals should be set to (pokemon) species_id with 2.
>    Don't worry I have added the bullet points with pseudo below. In short,
>    both your queries should be in the data.sql file and the PR
>    description with screenshots.
>
> Requirement
>
> [image: image]
> <https://user-images.githubusercontent.com/66476329/278327465-c0885738-00b2-4815-8971-b4f2f2ea25d9.png>
> Pseudo
>
>    1. Set the animals species_id = 1 where the name of the animals ends
>    with "%mon" which means the last three characters are mon. (It will update
>    the majority of the animals)
>    2. Set the rest of the animals with species_id = 2, where animals
>    species_id is null. (Those animals will get the species_id = 2 which does
>    contain the default value Null in the table)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ismayito/vet-clincic-database/pull/3#pullrequestreview-1699498838>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALMGCSKL4DHP436JTRVCFNLYBJJP5AVCNFSM6AAAAAA6QTWSB6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTMOJZGQ4TQOBTHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ismayito commented:
Please checkout my species table and see


On Thu, Oct 26, 2023 at 3:51 PM Mayito Ismail ***@***.***> wrote:

> Hello unless otherwise but i updated the SQL and everything is fine
>
>
> On Thu, Oct 26, 2023 at 3:28 PM Aakash Verma ***@***.***>
> wrote:
>
>> ***@***.**** requested changes on this pull request.
>> Hi @ismayito <https://github.com/ismayito>,
>>
>> Great work on making the changes requested by a previous reviewer 👏
>> You've done well implementing some of the requested changes, but some are
>> still not addressed yet.
>> STATUS: REQUIRE CHANGES 🔴
>>
>>
>> <https://camo.githubusercontent.com/7cb34c1f1afdb9d140a7831d0b08fbde98644d3dbb133fdcf4e69aeecaef2189/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784f47597959574e6b4d6a526b59324a6a596a67314d6a59314d324e6a59544e6b4e574e694e5451774e3245795a475a6d4e574d3559535a6c634431324d563970626e526c636d35686246396e61575a7a583264705a6b6c6b4a6d4e305057632f6d333836347242777742544b4d4162706e332f67697068792e676966>
>> Highlights
>>
>>    - Good *PR* title ✅
>>    - Descriptive *PR* description ✅
>>    - Implemented the features as per the previous code reviewer's
>>    request ✅
>>
>> Required Changes ♻️
>>
>> *Check the comments under the review.*
>> Optional suggestions
>>
>> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
>> the approval of this PR. However, I strongly recommend you take them into
>> account as they can make your code better.*
>> Cheers, and Happy coding!👏👏👏 ⚠️ WARNING ⚠️
>>
>> *Please, do not open a new Pull Request for re-reviews. You should use
>> the same Pull Request submitted for the first review, either valid or
>> invalid unless it is requested otherwise.*
>> Have any doubt ❓
>>
>> Feel free to leave any questions or comments in the PR thread if
>> something is not 100% clear.
>> Please, remember to tag @skyv26 <https://github.com/skyv26> in your
>> question so I can receive the notification. You can also follow me on the
>> below platforms
>> [image: @skyv26] <https://github.com/skyv26> [image: linkedin-skyv2022]
>> <https://www.linkedin.com/in/skyv2022/> [image: @vrma_aakash]
>> <https://twitter.com/vrma_aakash> [image: @skybrel]
>> <https://www.instagram.com/skybrel/> [image: @skybrel]
>> <https://www.frontendmentor.io/profile/skyv26>
>> ------------------------------
>>
>> *As described in the Code reviews limits policy
>> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
>> limited number of reviews per project (check the exact number in your
>> Dashboard). If you think the code review was unfair, you can request a
>> second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>> ------------------------------
>>
>> In data.sql
>> <https://github.com/ismayito/vet-clincic-database/pull/3#discussion_r1373085181>
>> :
>>
>> > +//All other animals are Pokemon
>> +UPDATE ANIMALS SET species_id=2 WHERE TRIM(NAME) LIKE '%mon';
>> +//Sam Smith owns Agumon.
>>
>>
>>    - Well done @ismayito <https://github.com/ismayito>, you did a great
>>    job by adding the query to update the animals but it seems like you removed
>>    the query for updating the species_id = 1 animals ends with %mon in
>>    name, now with an above-selected query, in the database all animals
>>    species_id will set to 2. But according to the project requirement
>>    that I also have added after this comment as a reference, you should set
>>    the animal species_id with 1 (Digimon), whose name ends with %mon and
>>    the rest of the animals should be set to (pokemon) species_id with 2.
>>    Don't worry I have added the bullet points with pseudo below. In short,
>>    both your queries should be in the data.sql file and the PR
>>    description with screenshots.
>>
>> Requirement
>>
>> [image: image]
>> <https://user-images.githubusercontent.com/66476329/278327465-c0885738-00b2-4815-8971-b4f2f2ea25d9.png>
>> Pseudo
>>
>>    1. Set the animals species_id = 1 where the name of the animals ends
>>    with "%mon" which means the last three characters are mon. (It will update
>>    the majority of the animals)
>>    2. Set the rest of the animals with species_id = 2, where animals
>>    species_id is null. (Those animals will get the species_id = 2 which does
>>    contain the default value Null in the table)
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ismayito/vet-clincic-database/pull/3#pullrequestreview-1699498838>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ALMGCSKL4DHP436JTRVCFNLYBJJP5AVCNFSM6AAAAAA6QTWSB6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTMOJZGQ4TQOBTHA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

skyv26 commented on data.sql:
```diff
@@ -15,3 +15,28 @@ VALUES('Charmander', '2020-02-08', 0, true, -11),
+
+//Modify your inserted animals so it includes the species_id value:
+//If the name ends in "mon" it will be Digimon
+//All other animals are Pokemon
+UPDATE ANIMALS SET species_id=2 WHERE TRIM(NAME) LIKE '%mon';
```
- [ ] As I mentioned previously in my last review, you need to add both queries where you are updating the animals with `species_id = 1`, name ends with `%mon` and the rest of the animals where the `species_id` is null, with `species_id = 2`. I agree that things are working on your end perfectly fine, but it doesn't mean that it works on my side too. We as code reviewer, run yours queries step by step to get the same results that you got at your end with the help of screenshots that you added. That's why I told you, not only to add the queries but also update your `screenshots` in the PR description. Please read my previous comment carefully if you have any doubts. Your previous code reviewers also mentioned that you need to update your PR description with the latest screenshots, which in your case are still missing.

### Please justify the below screenshot and the query that I selected below:

#### In Screenshots
![image](https://github.com/ismayito/vet-clincic-database/assets/66476329/5f0cee49-79b4-4f57-96c0-2aa0d3c6b110)

#### In actual your query is
```sql
UPDATE ANIMALS SET species_id=2 WHERE TRIM(NAME) LIKE '%mon';
```

**Kindly add the below queries to fulfil the requirements and update the screenshots in the PR description, which will help your next code reviewer to verify your work. It is very important to not miss any screenshots as per the milestone guidelines. After this you will left with 2 more review chances, so take your decision wisely and don't forget to implement the changes as I suggested.**


### Missing Queries (Look at the difference)
```suggestion
UPDATE ANIMALS SET species_id=1 WHERE TRIM(NAME) LIKE '%mon';
UPDATE ANIMALS SET species_id=2 WHERE species_id IS NULL;
```

### ⚠️ Your screenshots must match with your queries that you performed as per instructions, so add the updated screenshots in your PR. If you are still not convinced or not satisfied with the review, please use the second opinion form, which you can find in my review at the very end or [click here](https://airtable.com/apphXa7srZyRCpJVR/shrQAqnBwek5a0O0s)

skyv26 reviewed (changes requested):
## Hi @ismayito, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## STATUS: REQUIRE CHANGES 🔴 

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZDMzYmY2ZWU0MmEwMTJjZTczMGVhNDhlM2E2NWViYmQ0YzY5ZTBiNiZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/17FxSFyYNOgThnonDK/giphy.gif)

## Highlights
- N/A

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   <img src="https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white" alt="@skyv26" />
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   <img src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white" alt="linkedin-skyv2022" />
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   <img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white" alt="@vrma_aakash" />
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   <img src="https://img.shields.io/badge/Instagram-E4405F?style=for-the-badge&logo=instagram&logoColor=white" alt="@skybrel" />
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   <img src="https://user-images.githubusercontent.com/66476329/243158690-61eb1021-fbb6-464d-b841-a09c4caa156e.png" alt="@skybrel" />
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ismayito committed:
Fix review changes

mukhammadrizooff reviewed (approved):
Hi @ismayito, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

<img src="https://media4.giphy.com/media/sR9nilEznDoSlga667/giphy.gif"/> 

## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Screenshots updated fully✔️ 
- Descriptive `README` file ✔️
- Done all suggested changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
